edgelet/iotedged/src/lib.rs

```diff
@@ -183,6 +185,42 @@ pub struct Main {
     reactor: Core,
 }
 
+#[derive(Clone)]
+pub enum MaybeProxyClient {
```

Comment:
>MaybeProxyClient [](start = 9, length = 16)

I'll bet we could still write a test for this.

Reply:
Yes, definitely don't want to merge this until I've thought about (and written) tests.

Reply:
I've added tests for `MaybeProxyClient`.